src/typegraph/deno/src/runtimes/grpc.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+import { Func } from "../types.ts";
+import { runtimes } from "../wit.ts";
+import { Runtime } from "./mod.ts";
+
+export class GrpcRuntime extends Runtime {
+  constructor(protoFile: string, endpoint: string) {
+    const protoFileContent = new TextDecoder().decode(
+      Deno.readFileSync(protoFile),
+    );
+    const id = runtimes.registerGrpcRuntime({
+      protoFileContent,
```

Comment:
I would suggest you to give the file path in the sdk and use fs context to read its content, unormalized `cwd` can give issues or introduce sneaky bugs.

Reply:
Or better, treat proto files as artifacts just like how we do with DenoRuntime or PythonRuntime.

Reply:
But can we use fs context inside engine ? I'm not sure.

Reply:
Definitively, we need to discuss .XD